lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -101,8 +101,8 @@ def check_fails_too_quickly(status)
         end
       end
 
-      def run_vm
-        @machine.dump_config
+      def run_vm # rubocop:disable Metrics/AbcSize
```

Comment:
Rebase to current master. This method updated. 
If the AbcSize metric still will be big, update the Rubocop config according to the Slack discussion.